packages/phone-number-privacy/src/common/utils.ts

```diff
@@ -0,0 +1,15 @@
+import { Response } from 'firebase-functions'
+
+export enum ErrorMessages {
+  DATABASE_UPDATE_FAILURE = 'Failed to update database entry',
```

Comment:
Nice! Let's also include unique codes in the messages to make them easier to set up alerts for. How about `CELO_PNP_ERR_0X`